config/locales/en.yml

```
@@ -30,4 +30,7 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  base_title: "Pet Store"
+  sign_up_sucess: "Signed up successfully!"
+  sign_up_title: "Sign Up"
```

```suggestion
  sign_up:
    title: "Sign Up"
    create_sucess: "Signed up successfully!"
```
Nên chia theo cấp như thế này